cue/data/gen.cue

```diff
@@ -91,7 +92,7 @@ Family: scuemata.#Family & {
                     // _pv: { maj: int, min: int }
                     // The major and minor versions of the panel plugin for this schema.
                     // TODO 2-tuple list instead of struct?
-                    panelSchema: { maj: number, min: number }
+                    panelSchema?: { maj: number, min: number }
```

Comment:
for now, we don't have panelSchema yet, For the backward compatibility, i am wonder, should we put manually 0,0 after parsing the request.

Reply:
This does not seem like something that should ever be optional -- can we hardcode 0,0 if not set?

Reply:
oh never mind -- this is on the dashboard, and yes will often be missing!